aml-development/ozp-center
# refactor(carousel): Remove and Replace caroufredsel

Replace the current Carousel:
  This removes the current Carousel from center and creates a custom carousel.
  The purpose of this was to closely recreate the functionality of the current carousel.
  The carousel is React based instead of JQuery. It utilizes JQuery for animations.
  No new features have been added. Some minor functionality differences exist.

AMLNG-842

**Description**
The current carousel in use (caroufredsel) has limited some of what we've tried to do lately (recommender feedback and removing listings on the fly), so we're going to take some time to create our own.

**Acceptance Criteria**
* Similar behaviour to current carousel to maintain current user experience (click arrows to move slider/carousel, animated)
* Works with react style prop/state updates

**How to test code**
Pull this branch. You will need to wipe out your node modules and run npm install to get rid of Caroufredsel. After starting center, there are carousels on the main homepage and within the listing containing screenshots. Confirm that they function the same as the current carousel (caroufredsel).
